Setting up only `description` in `requestBody`

Is it possible to set only `description` in `requestBody` and allow rest to be autogenerated?

```go
func (tr TEERouter) RegisterRoutes(api huma.API) {
	operation := huma.Operation{
		Method:      http.MethodPost,
		Path:        "/tee",
		Summary:     "TEE",
		Description: "TEE description",
		RequestBody: &huma.RequestBody{
			Description: "My custom request schema",
		},
	}
	huma.Register(api, operation, tr.CalculateTEE)
}
```
is resulting in 
```yaml
/tee:
    post:
      description: TEE description
      requestBody:
        content: null
        description: My custom request schema
``` 
Trying to set it later
```go
func (tr TEERouter) RegisterRoutes(api huma.API) {
	operation := huma.Operation{
		Method:      http.MethodPost,
		Path:        "/tee",
		Summary:     "TEE",
		Description: "TEE description",
	}
	operation.RequestBody.Description = "My custom request schema"
	huma.Register(api, operation, tr.CalculateTEE)
}
```
is giving 
```
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x18 pc=0x842c69]
```

I need it to set it there for rapidapi.com to properly fill documentation data from provided OpenAPI file.

--- a/huma.go
+++ b/huma.go
@@ -572,30 +572,30 @@ func Register[I, O any](api API, op Operation, handler func(context.Context, *I)
 		hasInputBody = true
 		inputBodyIndex = f.Index
 		if op.RequestBody == nil {
-			required := f.Type.Kind() != reflect.Ptr && f.Type.Kind() != reflect.Interface
-			if f.Tag.Get("required") == "true" {
-				required = true
-			}
-
-			contentType := "application/json"
-			if c := f.Tag.Get("contentType"); c != "" {
-				contentType = c
-			}
-			hint := getHint(inputType, f.Name, op.OperationID+"Request")
-			if nameHint := f.Tag.Get("nameHint"); nameHint != "" {
-				hint = nameHint
-			}
-			s := SchemaFromField(registry, f, hint)
-
-			op.RequestBody = &RequestBody{
-				Required: required,
-				Content: map[string]*MediaType{
-					contentType: {
-						Schema: s,
-					},
-				},
-			}
+			op.RequestBody = &RequestBody{}
 		}
+
+		required := f.Type.Kind() != reflect.Ptr && f.Type.Kind() != reflect.Interface
+		if f.Tag.Get("required") == "true" {
+			required = true
+		}
+
+		contentType := "application/json"
+		if c := f.Tag.Get("contentType"); c != "" {
+			contentType = c
+		}
+		hint := getHint(inputType, f.Name, op.OperationID+"Request")
+		if nameHint := f.Tag.Get("nameHint"); nameHint != "" {
+			hint = nameHint
+		}
+		s := SchemaFromField(registry, f, hint)
+
+		op.RequestBody.Required = required
+
+		if op.RequestBody.Content == nil {
+			op.RequestBody.Content = map[string]*MediaType{}
+		}
+		op.RequestBody.Content[contentType] = &MediaType{Schema: s}
 
 		if op.BodyReadTimeout == 0 {
 			// 5 second default
@@ -615,8 +615,11 @@ func Register[I, O any](api API, op Operation, handler func(context.Context, *I)
 		if op.RequestBody == nil {
 			op.RequestBody = &RequestBody{
 				Required: true,
-				Content:  map[string]*MediaType{},
 			}
+		}
+
+		if op.RequestBody.Content == nil {
+			op.RequestBody.Content = map[string]*MediaType{}
 		}
 
 		contentType := "application/octet-stream"

--- a/huma_test.go
+++ b/huma_test.go
@@ -545,6 +545,42 @@ func TestFeatures(t *testing.T) {
 			Body: `{"name":"foo"}`,
 		},
 		{
+			Name: "request-body-description",
+			Register: func(t *testing.T, api huma.API) {
+				huma.Register(api, huma.Operation{
+					Method: http.MethodPut,
+					Path:   "/body",
+					RequestBody: &huma.RequestBody{
+						Description: "A description",
+					},
+				}, func(ctx context.Context, input *struct {
+					Body struct {
+						Name string `json:"name"`
+					}
+				}) (*struct{}, error) {
+					assert.Equal(t, "foo", input.Body.Name)
+					return nil, nil
+				})
+				// Note: the description should be set, but *also* the generated
+				// schema should be present since we didn't set it up ourselves.
+				b, _ := api.OpenAPI().Paths["/body"].Put.RequestBody.MarshalJSON()
+				assert.JSONEq(t, `{
+					"description": "A description",
+					"required": true,
+					"content": {
+						"application/json": {
+							"schema": {
+								"$ref": "#/components/schemas/Request"
+							}
+						}
+					}
+				}`, string(b))
+			},
+			Method: http.MethodPut,
+			URL:    "/body",
+			Body:   `{"name":"foo"}`,
+		},
+		{
 			Name: "request-body-nested-struct-readOnly",
 			Register: func(t *testing.T, api huma.API) {
 				type NestedStruct struct {

